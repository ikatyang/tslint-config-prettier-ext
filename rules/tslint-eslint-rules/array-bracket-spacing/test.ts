const arr = [ 'foo', 'bar' ];
const [ x, y ] = arr;
