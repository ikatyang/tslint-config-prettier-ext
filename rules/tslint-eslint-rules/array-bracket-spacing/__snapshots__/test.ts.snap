// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
const arr = [ 'foo', 'bar' ];
const [ x, y ] = arr;

======
const arr = [\\"foo\\", \\"bar\\"];
const [x, y] = arr;

>>>>>> after
"
`;

exports[`should fail before formatting 1`] = `
"ERROR: (array-bracket-spacing) /rules/tslint-eslint-rules/array-bracket-spacing/test.ts[1, 13]: There should be no space after \\"[\\"
ERROR: (array-bracket-spacing) /rules/tslint-eslint-rules/array-bracket-spacing/test.ts[1, 28]: There should be no space before \\"]\\"
ERROR: (array-bracket-spacing) /rules/tslint-eslint-rules/array-bracket-spacing/test.ts[2, 7]: There should be no space after \\"[\\"
ERROR: (array-bracket-spacing) /rules/tslint-eslint-rules/array-bracket-spacing/test.ts[2, 14]: There should be no space before \\"]\\"
"
`;
